firebase.json

```diff
@@ -1,5 +0,0 @@
-{
-  "react-native": {
-    "messaging_ios_auto_register_for_remote_messages": false
```

Comment:
Leftover from a bug in 6.X.X . Registering is done automatically by the firebase client